airflow/config_templates/config.yml

```diff
@@ -1704,6 +1704,14 @@
       type: float
       example: ~
       default: "15.0"
+    - name: fail_tis_with_missing_dag_file_interval
```

Comment:
I couldn't think of a better name here...

Reply:
Maybe something like `missing_dag_file_cleanup_interval`? We are basically cleaning those DAGs up, and failing tasks is a result of that (because the task can no longer be run).

Reply:
Apt!